contracts/cryptoblades.sol

```diff
@@ -209,7 +209,7 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
             // isCharacterOwner(char)
             // isWeaponOwner(wep) {
         fightModifierChecks(char, wep) {
-        require(fightMultiplier >= 1 && fightMultiplier <= 5);
+        require(fightMultiplier > 0);
```

Comment:
do not revert this change

Reply:
How do I avoid this I thought I merged correctly from the main project

Reply:
It looks like you did a lot of `--strategy=mine` merging, which as you can see completely reverted the last day or so of changes. I have no idea what setting you need to set to fix this though, but I would recommend re-merging your changes if possible so these overwrites don't happen.